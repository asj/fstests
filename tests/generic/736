#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright 2023 Meta
#
# FS QA Test No. generic/5736
#
# Run fscrypt on an encrypted directory
#

. ./common/preamble
_begin_fstest auto quick encrypt
echo

# Import common functions.
. ./common/filter
. ./common/encrypt

# real QA test starts here
_supported_fs generic
_require_scratch_encryption -v 2

_scratch_mkfs_encrypted &>> $seqres.full
_scratch_mount

dir=$SCRATCH_MNT/dir
mkdir $dir

_set_encpolicy $dir $TEST_KEY_IDENTIFIER
_add_enckey $SCRATCH_MNT "$TEST_RAW_KEY"

args=$(_scale_fsstress_args -p 4 -n 10000 -p 2 $FSSTRESS_AVOID -d $dir)
echo "Run fsstress $args" >>$seqres.full

$FSSTRESS_PROG $args >> $seqres.full

# success, all done
status=0
exit
