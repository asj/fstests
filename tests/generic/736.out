QA output created by 736

Added encryption key with identifier 69b2f6edeee720cce0577937eb8a6751
